### PR TITLE
Feat/handle attestors with no organization

### PR DESCRIPTION
**Pull request by MohammadPCh**
- #107 

## Summary by CodeRabbit

- **New Features**
	- Introduced a new organization entry "No Affiliation" for the database.
	- Added a constant `ZERO_UID` for enhanced configuration related to unique identifiers.
	- Enhanced project attestation handling with a modular approach for managing `AttestorOrganisation`.

- **Bug Fixes**
	- Improved error handling and logging in the `getOrCreateAttestorOrganisation` function.

- **Refactor**
	- Streamlined the `handleProjectAttestation` function for better maintainability and clarity.
	- Enhanced test suite for the `Organisation` model to ensure robust handling of duplicate entries.

### CODE REVIEW

File: src/controllers/utils/modelHelper.ts [hunk 1]
```diff
@@ -182,3 +184,74 @@ export const getProjectStats = async (
...
+  }
+
+  // Attempt to retrieve default organisation
+  let organisation;
+  try {
+    if (!NO_AFFILIATION_SCHEMA) {
+      ctx.log.error("NO_AFFILIATION_SCHEMA not set");
+      return undefined;
+    }
+    organisation = await ctx.store.get(Organisation, NO_AFFILIATION_SCHEMA);
```
Comment (aminlatifi) [inline src/controllers/utils/modelHelper.ts:222]: Is it necessary to put this part in a separate try catch?

File: db/migrations/1735226891776-AddNo Affiliation.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+module.exports = class AddNoAffiliation1735226891776 {
+  name = "AddNoAffiliation1735226891776";
+
+  async up(db) {
+    const SQUID_NETWORK = process.env.SQUID_NETWORK || "eth-sepolia";
+    if (SQUID_NETWORK !== "eth-sepolia") return;
+    // add organisation with name "No Affiliation" and schema id "0xb5b2e19821d6124101d31c7d5422a79402a0923a1ffaacbe3d8380688a18b7d1"
```
Comment (aminlatifi) [inline db/migrations/1735226891776-AddNo Affiliation.js:7]: Please get the no affiliation UID from env!

**Review by aminlatifi — CHANGES_REQUESTED** [review]

**Comment by MohammadPCh** [comment]
@aminlatifi you cannot use import in the js file.

File: src/controllers/utils/modelHelper.ts [hunk 3]
```diff
@@ -182,3 +184,52 @@ export const getProjectStats = async (
...
+  const key = `NO_AFFILIATION${attestor.id}`;
+
+  const newAttestorOrganisation = new AttestorOrganisation({
+    id: key,
+    attestor,
+    organisation,
+    attestTimestamp: timestamp,
+  });
+
+  await ctx.store.upsert(newAttestorOrganisation);
```
Comment (aminlatifi) [inline src/controllers/utils/modelHelper.ts:231]: I think we used upsert correctly, it shouldn't make issue if an identical item has been created before.

**Review by aminlatifi — APPROVED** [review]
LGTM! Thanks @MohammadPCh

**Pull request merged**